tests/test_connection_options.py

```diff
@@ -52,3 +53,29 @@ def test_mysql_explicit_ssl():
     backend = MySQLBackend("mysql://localhost/database", ssl=True)
     kwargs = backend._get_connection_kwargs()
     assert kwargs == {"ssl": True}
+
+
+def test_aiopg_pool_size():
+    backend = AiopgBackend("postgres+aiopg://localhost/database?min_size=1&max_size=20")
+    kwargs = backend._get_connection_kwargs()
+    assert kwargs == {"minsize": 1, "maxsize": 20}
+
+
+def test_aiopg_explicit_pool_size():
+    backend = AiopgBackend(
+        "postgres+aiopg://localhost/database", min_size=1, max_size=20
+    )
+    kwargs = backend._get_connection_kwargs()
+    assert kwargs == {"minsize": 1, "maxsize": 20}
+
+
+def test_aiopg_ssl():
+    backend = AiopgBackend("postgres+aiopg://localhost/database?ssl=true")
```

Comment:
Looks like we're expecting `postgres+aiopg` in some places and `postgresql+aiopg` in others.
I'd be okay with us accepting either or both.

Reply:
Right, yeah, it's a typo, but we only expect `postgresql+aiopg` as of now, at least there's only this mapping. it's not triggered in these tests as don't make any use of it. I'll update urls to avoid confusion.